src/classes/Frames.ts

```diff
@@ -0,0 +1,49 @@
+import { ErrorType } from "../types";
+
+export const getTFRFrames = async ({ sku }): Promise<ErrorType | void> => {
```

Comment:
This function name is ambiguous. Since we use frames in regards to both avatar and VTO, we would specify which one this is for. something like getVTOFrames would be better.